<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeControlsWidget</class>
 <widget class="QWidget" name="VolumeControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>189</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupCropping">
     <property name="title">
      <string>Volume Cropping</string>
     </property>
     <layout class="QVBoxLayout" name="croppingVBox">
      <item>
       <layout class="QHBoxLayout" name="axialRangeLayout">
        <item>
         <widget class="QLabel" name="axialMinLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="RangeSlider" name="axialRangeSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="axialMaxLabel">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="saggitalRangeLayout">
        <item>
         <widget class="QLabel" name="saggitalMinLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="RangeSlider" name="saggitalRangeSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="saggitalMaxLabel">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="coronalRangeLayout">
        <item>
         <widget class="QLabel" name="coronalMinLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="RangeSlider" name="coronalRangeSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="coronalMaxLabel">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonGroupLayout">
        <item>
         <widget class="QPushButton" name="presetFullVolume">
          <property name="text">
           <string>Full Volume</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="presetCenterROI">
          <property name="text">
           <string>Center ROI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="presetReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="slicePlaneCheckBox">
        <property name="text">
         <string>Show Slice Planes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RangeSlider</class>
   <extends>QSlider</extends>
   <header>RangeSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
